spring-cloud-stream/src/main/java/org/springframework/cloud/stream/function/FunctionInvoker.java

```diff
@@ -141,10 +143,19 @@
 	}
 
 	private void onError(Throwable t, Message<I> originalMessage) {
-		if (this.errorChannel != null) {
-			ErrorMessage em = new ErrorMessage(t, (Message<?>) originalMessage);
+		String inputDestinationName = functionProperties.getInputDestinationName();
+		BindingProperties bindingProperties = functionProperties.getBindingServiceProperties().getBindings().get(inputDestinationName);
+		String destinationName = bindingProperties.getDestination();
+		String groupName = bindingProperties.getGroup();
+		String bindingErrorChannelName = destinationName + "." + groupName + ".errors";
```

Comment:
Perhaps change `AbstractMessageChannelBinder.errorsBaseName()` to `public static` and call it here?

Reply:
There is a `C` type for consumer properties which is not really used, but I would need to change it to Object or something. . . I'll keep it as is for now since there may be more refactoring soon

Reply:
ok